src/compile/repeat.ts

```diff
@@ -40,7 +40,8 @@ export class RepeatModel extends BaseConcatModel {
     // cross product
     for (const rowField of row) {
       for (const columnField of column) {
-        const name = (rowField ? '_' + rowField : '') + (columnField ? '_' + columnField : '');
+        const name =
+          (rowField ? `__repeat-row_${rowField}` : '') + (columnField ? `__repeat-column_${columnField}` : '');
```

Comment:
@domoritz @arvind 

Do we have a method to escape rowField so that it's a valid CSS name? (Our spec name must be a valid CSS name.)

Reply:
I don't know.

Reply:
We have `varName` which I think should produce valid CSS names.